gtsfm/densify/mvs_math.py

```diff
@@ -0,0 +1,41 @@
+"""MVS math methods for gtsfm
+
+Authors: Ren Liu
+"""
+import math
+import numpy as np
+
+from gtsfm.utils.geometry_comparisons import angle_between_vectors
+
+
+def piecewise_gaussian(
+    p_a: np.ndarray, p_b: np.ndarray, theta_0: float = 5, sigma_1: float = 1, sigma_2: float = 10
+) -> float:
+    """Evaluate the similarity of measurements for the same track p in view a and view b.
+
+    This piecewise Gaussian function will outputs a float score to show the evaluation result of the similarity.
```

Comment:
Two minor typos:
1. This piecewise Gaussian function ~will outputs~ outputs a float score
2. ~calcualted~ calculated.

Reply:
Float score seems like an implementation detail. Maybe a score in the range [0,1], if that’s true?